src/cache/redisCache.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { CACHE_MANAGER } from '@nestjs/cache-manager';
+
+@Injectable()
+export class RedisCacheService {
+  constructor(@Inject(CACHE_MANAGER) private readonly cache: Cache) {}
+
+  async get(key: string): Promise<any> {
+    return await this.cache.get(key);
+  }
+
+  async set(key: string, value: any) {
```

Comment:
Missing typedef for a return value. It's really easy to introduce errors when you don't specify the types of returned values, [as TS will infer the type from the function](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAJwKZRMsBnAKqgDygAoBKRAbwChEV1MkByAQ2wBNGBuKgXyqtCRYCOhix5CUAExlKNUQ0qIokgFyIW7DT258qEBNiiIAbswA2ARkQBeBePxEy3APQvatAHoB+fgZzGZuZStvY4jtLOVG4eiD78QZYAdFBwADJwAO6oyADCrKhRQVIp6Vk5+diFpJxAA) (it's not always the type you actually want to be returned).

```suggestion
  async set(key: string, value: any): Promise<void> {
```